tests/EvmAcceptanceTests/test/GasEstimations.js

```diff
@@ -28,6 +37,16 @@ describe("Gas estimation with web3.js", function () {
     it("Should return proper gas estimation [@transactional]", async function () {
       const gasAmount = await contract.methods.installChild(123).estimateGas();
       expect(gasAmount).to.be.at.least(CREATE2_MIN_GAS);
+
+      const result = await contract.methods
+        .installChild(123)
+        .send({gasLimit: 420000000, from: web3_helper.getPrimaryAccountAddress()});
+
+      expect(result).to.be.not.null;
+      const actualGas = result.gasUsed;
+
+      expect(actualGas).to.be.at.least(gasAmount * 0.9);
```

Comment:
this test now makes sure the gas estimated is within 10% of what it actually costs to use